images/timoni/tests/01-help.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+docker run --rm $IMAGE_NAME help
```

Comment:
Any reason not to just inline this?

I've been inlining more one-liners.

Reply:
Done!